protocol/messenger_communities.go

```diff
@@ -137,6 +138,17 @@ func (m *Messenger) JoinCommunity(ctx context.Context, communityID types.HexByte
 		return nil, err
 	}
 
+  communitySettings := params.CommunitySettings{
+    CommunityID: communityID.String(),
+    MessageArchiveFetchingEnabled: true,
+    MessageArchiveSeedingEnabled: false,
+  }
```

Comment:
These are the defaults when joining a community.

If could also introduce a `GetDefaultCommunitySettings` API if you prefer.